src/Tasks/Microsoft.NET.Build.Tasks/msbuildExtensions/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.ConflictResolution.targets

```diff
@@ -0,0 +1,45 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Choose>
+    <!-- Allow completely disabling the conflict resolution targets -->
+    <When Condition="'$(DisableHandlePackageFileConflicts)' == 'true'" />
+    <!-- SDK projects have built in conflict resolution that understands assets/deps file -->
+    <When Condition="'$(UsingMicrosoftNETSdk)' == 'true'" />
+    <When Condition="'$(ResolveNuGetPackages)' == 'true' and Exists('$(ProjectLockFile)')">
+      <!-- Non-SDK using project.json or PackageReference, run after references are read from the lock/assets file -->
+      <PropertyGroup>
+        <HandlePackageFileConflictsAfter>ResolveNuGetPackageAssets</HandlePackageFileConflictsAfter>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <!-- No specific NuGet targets to sequence after, run before targets that consume references -->
+      <PropertyGroup>
+        <ResolveAssemblyReferencesDependsOn>$(ResolveAssemblyReferencesDependsOn);HandlePackageFileConflicts</ResolveAssemblyReferencesDependsOn>
+        <PrepareResourcesDependsOn>HandlePackageFileConflicts;$(PrepareResourcesDependsOn)</PrepareResourcesDependsOn>
+      </PropertyGroup>
+    </Otherwise>
+  </Choose>
+  
+  <UsingTask TaskName="ResolvePackageFileConflicts" AssemblyFile="$(MicrosoftNETBuildExtensionsTasksAssembly)" />
+  <Target Name="HandlePackageFileConflicts" AfterTargets="$(HandlePackageFileConflictsAfter)" DependsOnTargets="$(HandlePackageFileConflictsDependsOn)">
+    <HandlePackageFileConflicts References="@(Reference)"
+                                ReferenceCopyLocalPaths="@(ReferenceCopyLocalPaths)"
+                                PlatformManifests="@(PackageConflictPlatformManifests)"
```

Comment:
Do the `PlatformManifests` and `PreferredPackages` parameters apply here?  Don't those only apply when targeting .NET Core or .NET Standard, in which the SDK target would be used instead?